runBottlecap.sh

```diff
@@ -10,8 +18,8 @@ echo -e 'export const handler = async () => {\n\tconsole.log("Hello world!");\n}
 docker cp "/tmp/index.mjs" "${docker_name}:/var/task/index.mjs"
 docker start "${docker_name}"           
 docker exec "${docker_name}" mkdir -p /opt/extensions
-docker cp bottlecap/target/debug/bottlecap "${docker_name}:/opt/extensions/bottlecap"
```

Comment:
did you test on codespace as well? not sure that codespace arch is aarch64?

Reply:
Nope that's totally my fault, thanks for noting this. Fixed!